uint64 timestamp
waterlinked_a50/PositionReport report